### PR TITLE
update docker-gcplogs-driver links

### PR DESCRIPTION
**Pull request by tswast**
When investigating #220, I found a few links that needed updating.

### CODE REVIEW

File: tutorials/docker-gcplogs-driver/index.md [hunk 1]
```diff
@@ -53,35 +54,30 @@ automatic system updates.
...
-        docker run -d --name mysite --log-driver=gcplogs -p 80:80 nginx
-
-    You can specify additional options with the `--log-opt` command-line
-    argument. This command tells Docker to also log the command that the container
-    was started with.
-
         docker run -d \
             --name mysite \
             --log-driver=gcplogs \
-            --log-opt gcp-log-cmd=true \
```
Comment (tswast) [inline tutorials/docker-gcplogs-driver/index.md]: In my testing today, I couldn't get this to actually have any effect, so I've removed it.

File: tutorials/docker-gcplogs-driver/index.md [hunk 2]
```diff
@@ -196,7 +192,7 @@ you can skip this section.*
 From [Google Cloud Shell](/shell/docs/quickstart) or a development machine
 where you have [installed and initialized the Google Cloud SDK](/sdk/docs/),
 use the [gcloud compute intances
-add-metadata](https://cloud.google.com/sdk/gcloud/reference/compute/instances/add-metadata)
+add-metadata](/sdk/gcloud/reference/compute/instances/add-metadata)
```
Comment (ToddKopriva) [inline tutorials/docker-gcplogs-driver/index.md]: It's best to keep the full URL. The relative URL will work in the context of cloud.google.com, but some folks also read the tutorials here on GitHub.

File: tutorials/docker-gcplogs-driver/index.md [hunk 3]
```diff
@@ -226,6 +222,5 @@ command to add the `user-data` key to your instance.
 
 - [Install the Stackdriver Logging Agent to stream system logs to Stackdriver
   Logging](/logging/docs/agent/installation).
-- [Try out some other Docker-related community
-  tutorials](/community/tutorials/?q=Docker).
+- [Try out some other DevOps tutorials](/docs/tutorials/#devops).
```
Comment (ToddKopriva) [inline tutorials/docker-gcplogs-driver/index.md]: same comment about absolute versus relative URL

File: tutorials/docker-gcplogs-driver/index.md [hunk 4]
```diff
@@ -226,6 +222,5 @@ command to add the `user-data` key to your instance.
 
 - [Install the Stackdriver Logging Agent to stream system logs to Stackdriver
   Logging](/logging/docs/agent/installation).
```
Comment (ToddKopriva) [inline tutorials/docker-gcplogs-driver/index.md]: same comment about absolute versus relative URL

File: tutorials/docker-gcplogs-driver/index.md [hunk 5]
```diff
@@ -226,6 +222,5 @@ command to add the `user-data` key to your instance.
 
 - [Install the Stackdriver Logging Agent to stream system logs to Stackdriver
```
Comment (ToddKopriva) [inline tutorials/docker-gcplogs-driver/index.md]: The page that this points to uses a lowercase 'a' in 'agent'.

**Pull request merged**